### DISCUSSION TITLE
Visibility in IOS

### DISCUSSION THREAD

**Discussion by jmiguels** [espressif/arduino-esp32#9872] [Question - Community Help]
I am using this library to communicate with Android and IOS. No problem with Android. IOS is not working - it is not detected.
I am using the BluetoothSerial example "SerialToSerialBT.ino".
Can anybody help?

**Comment by me-no-dev** [comment]
iOS does not support SPP over classic BT. You need BLE

**Comment by jmiguels** [comment]
Thanks for your answer. Can you sugest a BLE library that can be used for serial point to point in arduino IDE?
